index.js

```diff
@@ -52,6 +53,7 @@ module.exports = function(config) {
     (!config.cloudwatchNamespace && config.cloudwatchStackname)) throw new Error('both cloudwatchNamespace and cloudwatchStackname must be configured');
 
   config.maxProcessTime = config.maxProcessTime || 3e5;
+  config.minProcessTime = config.minProcessTime || 0;
```

Comment:
I find even 500ms pretty low for a default option. I'd lean more towards 1000ms, not knowing if/how many other consumers are reading from the same stream.